pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -287,7 +287,10 @@
       }
     },
     "Microsoft.Win32.SystemEvents": {
-      "InboxOn": {},
+      "InboxOn": {
+        "netcoreapp2.0": "4.0.0.0",
```

Comment:
I don’t think it should be inbox in netcoreapp2.0 as it never shipped inbox there. It will be inbox in 3.0, not even 2.1